src/wlapplication.cc

```diff
@@ -1293,14 +1236,26 @@ void WLApplication::handle_commandline_parameters() {
 			exit(1);
 		}
 	}
+
+	if (commandline_.count("language")) {
```

Comment:
should me not check this first?

Reply:
We need to setup the datadir/localedir before having access to the translation. So this is the earliest point, where the language can be changed.